Source/Scene/PointCloud3DTileContent.js

```diff
@@ -1203,10 +1224,18 @@ define([
 
         this._mode = frameState.mode;
 
+        var context = frameState.context;
+
         // update clipping planes
+        var clippingPlanes = this._tileset.clippingPlanes;
+        var modelViewChanged = context.uniformState._modelViewDirty || modelMatrixChanged;
```

Comment:
While convenient, I don't think it is safe to use `_modelViewDirty`. Maybe we should always assume the view is dirty.